pkg/oss/nodeserver.go

```diff
@@ -392,6 +394,9 @@ func (o *Options) MakeMountOptionsAndAuthConfig(m metadata.MetadataProvider, vol
 	}
 	mountOptions = append(mountOptions, authOptions...)
 
+	defaultOSSFSArgs := os.Getenv("DEFAULT_OSSFS_ARGS")
+	mountOptions = append(mountOptions, strings.Split(defaultOSSFSArgs, ",")...)
```

Comment:
Maybe this should belongs to `func (f *fuseOssfs) AddDefaultMountOptions(options []string) []string`?

Split by comma can be problematic as we cannot include comma in the option itself. But maybe fine, since we already doing this in `parseOtherOpts`.